bait-files/src/main/webapp/submission.js

```diff
@@ -1,10 +1,270 @@
-function fetchBlobstoreUrlAndShowImage() {
+function fetchBlobstoreUrl() {
   fetch('/blobstore-upload')
       .then(response => {
         return response.text();
       })
       .then((imageUploadUrl) => {
-        const displayImage = document.getElementById('submission-form');
-        displayImage.action = imageUploadUrl;
+        const submissionForm = document.getElementById('submission-form');
+        submissionForm.action = imageUploadUrl;
       });
 }
+
+let map;
```

Comment:
I'm not sure this Map stuff should be in the same JS. Can we move it to a Map.js (or something)?

Reply:
moved to a new file called Map.js